FirebaseStorageSwift.podspec

```diff
@@ -14,7 +14,7 @@ Firebase Storage provides robust, secure file uploads and downloads from Firebas
 
   s.source                  = {
     :git => 'https://github.com/Firebase/firebase-ios-sdk.git',
-    :tag => 'StorageSwift-' + s.version.to_s
+    :tag => 'CocoaPods-' + s.version.to_s
```

Comment:
Just realized for here and elsewhere - we'll need to make sure we also create the `CocoaPods-7.Y.Z-beta` tag each release too

Reply:
SG, thanks for looking into it!